docs/book/src/tasks/experimental-features/machineset-preflight-checks.md

```diff
@@ -1,12 +1,57 @@
 # Experimental Feature: MachineSetPreflightChecks (alpha)
 
-The `MachineSetPreflightChecks` feature can provide additional safety while creating new Machines for a MachineSet.
+The `MachineSetPreflightChecks` feature can provide additional safety while creating new Machines and remediating existing unhealthy Machines of a MachineSet.
 
 **Feature gate name**: `MachineSetPreflightChecks`
 
 **Variable name to enable/disable the feature gate**: `EXP_MACHINE_SET_PREFLIGHT_CHECKS`
 
-The following preflight checks are performed when the feature is enabled:
-* ControlPlaneIsStable
-* KubeadmVersionSkew
-* KubernetesVersionSkew
\ No newline at end of file
+## Supported PreflightChecks
+
```

Comment:
I think Fabrizio was also mentioning that we should explain why each of those checks is important (aka a short concise description for each what problems might occur if they are not run / skipped)

Reply:
Not sure if we need a description per preflight check as the problems are similar:
* we would create Machines which versions outside of the official supported version range => it's unclear if the Machine creation woudl work => can lead to massive Machine churn
* control plane stable => as long as the control plane is not stable it's not possible to ensure the join works as it's unclear against which kube-apiserver version we join
* ... maybe more

Reply:
I added a small block at the top of the page to provide context on the MachineSet Preflight checks (pulled form the ms preflight checks issue). 

PTAL.